car-dealer/carRouter.js

```diff
@@ -0,0 +1,26 @@
+const require('express'); 
+const db = require('../knexfile')
+
+// setting up this particular route
+const router = express.Router(); 
+
+
+router.get('/', (req, res) => {
+    db('car-dealer')
+        .then(car => {
+            res.status(200).json(car); 
+        })
+        .catch(err => {
+            res.status(500).json(err); 
+        })
+})
+
+router.post('/', (req, res => {
```

Comment:
good work, maybe try and make a check for if the information got passed next time